### ISSUE TITLE
Unhandled element clipPath

### ISSUE THREAD

**Issue by hash404** [dnfield/flutter_svg#94]
I get this error from displaying an SVG (emoji_u1f600.svg from noto-emoji https://github.com/googlei18n/noto-emoji/tree/master/svg):

```
 flutter: unhandled element clipPath; Picture key: AssetBundlePictureKey(..)
```

Here is the incorrect rendering:

<img width="307" alt="emoji_u1f600" src="https://user-images.githubusercontent.com/36620537/50530821-a29f8600-0b55-11e9-9c16-2833a4de1d4f.png">

For reference this is the SVG

```svg
<svg enable-background="new 0 0 128 128" viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><symbol viewBox="-64.5 -64.5 129 129"><g fill="none" stroke="#4285F4" stroke-miterlimit="10"><rect x="-64" y="-64" width="128" height="128"/><path d="M36.95,37.82C27.32,46.32,14.2,51,0,51 c-14.27,0-27.39-4.62-36.96-13.01C-47.45,28.79-53,15.65-53,0c0-15.58,5.55-28.69,16.04-37.92C-27.36-46.35-14.24-51,0-51 c14.17,0,27.29,4.71,36.95,13.25C47.45-28.45,53-15.4,53,0C53,15.47,47.45,28.55,36.95,37.82z"/><path d="m0 55c-29.59 0-57-19.01-57-55 0-35.8 27.41-55 57-55s57 19.69 57 55c0 35.51-27.41 55-57 55z"/><path d="M0-43c-12.29,0-23.54,3.94-31.68,11.09 C-40.39-24.25-45-13.21-45,0c0,29.7,22.6,43,45,43c21.67,0,45-13.46,45-43S21.67-43,0-43L0-43z"/><line x1="-.01" x2="-.01" y1="51" y2="-51"/><line x1="-16" x2="-16" y1="48.95" y2="-48.93"/><line x1="15.99" x2="15.99" y1="48.91" y2="-48.93"/><line x1="53" x2="-53" y1=".08" y2=".08"/></g></symbol><linearGradient id="e" x1="64" x2="64" y1="19.717" y2="107.7" gradientUnits="userSpaceOnUse"><stop stop-color="#FEE133" offset=".5"/><stop stop-color="#FEDE32" offset=".6118"/><stop stop-color="#FCD32F" offset=".7076"/><stop stop-color="#FAC12B" offset=".7975"/><stop stop-color="#F7A924" offset=".8834"/><stop stop-color="#F6A323" offset=".9"/></linearGradient><path d="M64,119.89C36.07,119.89,6,102.4,6,64S36.07,8.11,64,8.11c15.48,0,29.81,5.12,40.36,14.43 C115.9,32.72,122,47.06,122,64c0,16.86-6.1,31.17-17.64,41.39C93.78,114.74,79.45,119.89,64,119.89z" fill="url(#e)"/><path d="m64 10.11c14.99 0 28.86 4.95 39.03 13.93 11.1 9.79 16.97 23.61 16.97 39.96 0 16.27-5.87 30.07-16.97 39.89-10.21 9.03-24.07 14-39.03 14-15.04 0-28.9-4.91-39.04-13.82-11.1-9.75-16.96-23.61-16.96-40.07 0-16.54 5.86-30.42 16.96-40.15 10.11-8.86 23.97-13.74 39.04-13.74m0-4c-31.15 0-60 20-60 57.89 0 37.68 28.85 57.89 60 57.89s60-20.73 60-57.89c0-37.37-28.85-57.89-60-57.89z" fill="#EB8F00"/><defs><path id="d" d="m63.47 62.53c-15.71 0-30.31-1.32-42.49-3.57-3.02-0.56-5.69 2.1-5.08 5.11 4.04 19.81 23.55 36.08 47.55 36.08 26.27 0 43.94-16.8 47.61-36.15 0.57-2.99-2.1-5.6-5.09-5.04-12.18 2.25-26.78 3.57-42.5 3.57z"/></defs><clipPath id="a"><use xlink:href="#d"/></clipPath><path d="m63.47 62.53c-15.71 0-30.31-1.32-42.49-3.57-3.02-0.56-5.69 2.1-5.08 5.11 4.04 19.81 23.55 36.08 47.55 36.08 26.27 0 43.94-16.8 47.61-36.15 0.57-2.99-2.1-5.6-5.09-5.04-12.18 2.25-26.78 3.57-42.5 3.57z" clip-path="url(#a)" fill="#242424"/><path d="M116.72,61.38V46.49H10.24v14.89C26.61,71,44.6,76.35,63.48,76.35 S100.34,71,116.72,61.38z" clip-path="url(#a)" fill="#fff"/><path d="m63.48 86.42c-11.78 0-22.05 6.42-27.53 15.95 7.12 3.89 16.83 6.29 27.53 6.29s20.41-2.4 27.53-6.29c-5.48-9.53-15.75-15.95-27.53-15.95z" clip-path="url(#a)" fill="#ED7770"/><g clip-path="url(#a)"><defs><path id="c" d="m10.24 46.49v14.89c16.38 9.62 34.36 14.97 53.24 14.97s36.86-5.35 53.24-14.97v-14.89h-106.48zm53.24 39.93c-11.78 0-22.05 6.42-27.53 15.95 7.12 3.89 16.83 6.29 27.53 6.29 10.71 0 20.41-2.4 27.53-6.29-5.48-9.53-15.75-15.95-27.53-15.95z"/></defs><clipPath id="b"><use xlink:href="#c"/></clipPath><g clip-path="url(#b)"><path d="m106.79 62.88c0.16 0 0.26 0.11 0.29 0.14 0.08 0.1 0.07 0.18 0.06 0.23-1.69 8.9-6.63 17.06-13.92 22.97-8.01 6.49-18.3 9.92-29.76 9.92-10.51 0-20.73-3.47-28.8-9.76-7.76-6.06-13.03-14.27-14.83-23.11-0.01-0.05-0.03-0.13 0.06-0.23 0.03-0.03 0.13-0.15 0.29-0.15 0.03 0 0.05 0 0.08 0.01 12.85 2.38 27.8 3.64 43.22 3.64 15.43 0 30.38-1.26 43.24-3.64 0.02-0.02 0.04-0.02 0.07-0.02m0-4c-0.27 0-0.54 0.02-0.81 0.08-12.18 2.26-26.79 3.58-42.51 3.58-15.71 0-30.31-1.32-42.49-3.57-0.27-0.05-0.54-0.08-0.81-0.08-2.67 0-4.83 2.44-4.27 5.18 4.04 19.81 23.55 36.08 47.55 36.08 26.27 0 43.94-16.8 47.61-36.15 0.52-2.72-1.63-5.12-4.27-5.12z" fill="#EB8F00"/></g></g><path d="m82.74 30.19c4.19 0 8 3.54 8 9.42s-3.81 9.42-8 9.42-8-3.54-8-9.42 3.81-9.42 8-9.42z" fill="#242424"/><path d="m45.2 30.19c-4.19 0-8 3.54-8 9.42s3.81 9.42 8 9.42 8-3.54 8-9.42-3.81-9.42-8-9.42z" fill="#242424"/></svg>
```

**Comment by dnfield** [comment]
The previously mentioned commit wouldn't handle nested groups properly.  Fixed in 6f3636c

**Comment by dnfield** [comment]
Published as v0.8.3
